db/migrate/20220523173933_create_transactions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateTransactions < ActiveRecord::Migration[7.0]
+  def change
+    create_table :transactions do |t|
+      t.string :name
+      t.float :amount
+      t.references :user, null: false, foreign_key: true
+      t.references :category, null: false, foreign_key: true
+
```

Comment:
- [ ] Kindly make your foreign key have indexes. You can run another migration file just for indexes:

```
class AddIndexToTable < ActiveRecord::Migration
  def change
    add_index :table, :user_id
  end
end
```

Reply:
same as the above comment